apps/consulting/posts/will-python-be-1-forever.md

```diff
@@ -0,0 +1,35 @@
+---
+title: 'Will Python Be #1 Forever?'
+published: June 1, 2020
+author: troy-bailey
+description: >
+  Based on the data collected by http://pypl.github.io/, over the last 5 years, PHP and Java declined in popularity while Python’s popularity rose. Sometime in 2018, Python became the most popular language. But Python probably won’t be #1 forever. If history is any indication, no language can maintain the premier position in language popularity indefinitely. I’m sure there were many who believed that Java would never be replaced as #1.
+category: [Open Source Software, PyData Ecosystem]
+featuredImage:
+  src: /posts/will-python-be-1-forever/python-1-img-1.jpg
+  alt: ''
+hero:
+  imageSrc: /posts/hero-paris.webp
+  imageAlt: 'Data visualization of Paris city'
+---
+
+Based on the data collected by [PYPL][pypl page] and shown in the chart below,
+over the last 5 years, PHP and Java declined in popularity while Python’s
+popularity rose. Sometime in 2018, Python became the most popular language. But
+Python probably won’t be #1 forever. If history is any indication, no language
+can maintain the premier position in language popularity indefinitely. I’m sure
+there were many who believed that Java would never be replaced as #1.
+
+![](/posts/will-python-be-1-forever/python-1-img-1.jpg)
+
+_([Image source: PYPL][pypl page])_
+
+But that certainly does not mean that Python is going away completely. Virtually
+none of the languages that have come to prominence in the past have disappeared
+yet. They continue on as legacy code. Looking at the bottom right of the same
```

Comment:
The text leads the reader through the plot, but the languages described in the text are not present in the plot.  Consider replacing the plot, adding a plot, or modifying the text.

Reply:
That looks a lot better.  Thanks!